Assets/Scripts/Map/MapManager.cs

```diff
@@ -19,12 +19,19 @@ public class MapManager : MonoBehaviour
         [SerializeField]
         private string _seed;
 
+        [SerializeField]
+        private GameObject _playerPrefab;
```

Comment:
I suggest we keep the player spawning separate from the map generation.
All we need to be able to do is query the map manager for the spawn room.

Reply:
As for now there is no class for entities management so this is just a temporary solution
That part should come more in depth when we will handle enemies spawning